### PR TITLE
Added Manish's info

### PR DESCRIPTION
**Pull request by Manice18**

### CODE REVIEW

File: Manice.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Manish Saha
```
Comment (adviksinghania) [inline Manice.md]: The heading is supposed to have 3 hashes, making it an H3 heading and not the main H1.

**Review by adviksinghania — CHANGES_REQUESTED** [review]

**Comment by Manice18** [comment]
Done @adviksinghania

**Comment by Manice18** [comment]
Please merge this one @adviksinghania . And please comment if there are any more mistakes.

**Review by adviksinghania — APPROVED** [review]
Nice

**Pull request merged**